cmd/skopeo/sync.go

```diff
@@ -562,6 +580,7 @@ func (opts *syncOptions) run(args []string, stdout io.Writer) error {
 		DestinationCtx:                        destinationCtx,
 		ImageListSelection:                    imageListSelection,
 		OptimizeDestinationImageAlreadyExists: true,
+		ForceManifestMIMEType:                 manifestType,
```

Comment:
any man page changes?

Reply:
Also `completions/bash/skopeo` (which seems to be missing most of `sync` options).

Reply:
@mtrmac would adding completions for `sync` (which don't exist today) be out of scope for this PR?

Reply:
No you should add them.

Reply:
Please do add a completion for the new `--format` option.

I don’t think it would be fair to block this PR on adding full completion support for the pre-existing options, though we certainly wouldn’t reject that work :)